operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -125,8 +141,15 @@ and managed in Vault.
 
 After completing the previous procedures, apply the configuration to the NCNs
 by running NCN personalization with [CFS](../configuration_management/Configuration_Management.md).
+This can be accomplished by running the `apply_csm_configuration.sh` script, or
+running the steps manually. For more information on the script, see
+[Automatically Apply CSM Configuration to NCNs](#auto_apply_csm_config)
```

Comment:
Nit: should have a period at the very end of the line.

Reply:
Fixed